src/lib.rs

```diff
@@ -16,8 +16,8 @@ pub use accumulators::{
     Accumulator, AccumulatorError,
 };
 
-mod binary_tree;
-pub use binary_tree::Height;
+pub mod binary_tree; // TODO: this as private in production
```

Comment:
This is not supposed to be made public. Rather use the tree construction API that is mentioned in the docs

Reply:
Perhaps we should move the benchmarks into `src`?

Reply:
The same thing happens when writing tests outside of `src`, which is often done for integration tests

Reply:
Let's chat on Monday at standup

Reply:
Sure. I realise I am mistaken – re-exporting is the obvious solution, which I have noticed you do everywhere else. I've just run into a similar problem before with integration tests